### DISCUSSION TITLE
⭐️ Starstruck Badge Exchange ⭐️

### DISCUSSION THREAD

**Discussion by Rantoryu** [Schweinepriester/achievement-testarea#34] [Q&A]
Help me get starstruck badge, I will ⭐️**star**⭐️ your repo back - provide link in the comment please!
https://github.com/Rantoryu/Personal_Website 

_**Note:** You can see who's actually starred your repo (e.g. "returned the favor") by appending /stargazers to the end of your repo's URL. For example: https://github.com/Rantoryu/Personal_Website/stargazers_

**Comment by pulkitxm** [comment]
star my  [repo](https://github.com/Pulkitxm/Delta-Apna-College), I'll do it for yours
 [https://github.com/Pulkitxm/Delta-Apna-College](https://github.com/Pulkitxm/Delta-Apna-College)
Have a nice Day !

**Reply by Rantoryu** [reply]
done :)

**Reply by pulkitxm** [reply]
Done bro thanks!

**Reply by fabricio-fn** [reply]
@rizwan3d @Pulkitxm  Done, https://github.com/fabricio-fn/ChatGpt-API

**Reply by 1wos** [reply]
@fabricio-fn I stared your repo ! Please star my repo - 👍🏻 https://github.com/sommizzu/ASF-DataAnalysis

**Reply by AdityaNow** [reply]
Done 😃 (feel free to star my repo) https://github.com/AdityaNow/NavtorAssignment

**Comment by zoelesv** [comment]
Nice repo! I gave it a ⭐️. Check out my repo and help me get my first ⭐️ as well! https://github.com/zoelesv/Smathchat

**Reply by fabricio-fn** [reply]
Done!

**Reply by 1wos** [reply]
I stared your repo ! Please star my repo - 👍🏻 https://github.com/sommizzu/ASF-DataAnalysis

**Reply by fabricio-fn** [reply]
@sommizzu  Done!

**Reply by AdityaNow** [reply]
@zoelesv Done 😃 ( Please feel free to star my repo) https://github.com/AdityaNow/NavtorAssignment

**Comment by ternera** [comment]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by Mattral** [comment]
Done!

Please mine too 
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by AdityaNow** [reply]
@Mattral Done 😃 ( Please feel free to star my repo) https://github.com/AdityaNow/NavtorAssignment

**Comment by the1Riddle** [comment]
hello @Rantoryu

i just stared your repo's, can you please star mine too, link: [installation tools](https://github.com/the-1Riddle/packageInstallationHub)

**thanks**

**Reply by fabricio-fn** [reply]
@the1Riddle Done!

Please mine too
https://github.com/fabricio-fn/ChatGpt-API

**Reply by the1Riddle** [reply]
thanks @fabricio-fn , am done with that.

**Reply by the1Riddle** [reply]
> Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

hello @ternera i have stared your repo, please star mine too.

**Comment by AdityaNow** [comment]
Greetings, can you please star my repo too, link: [https://github.com/AdityaNow/NavtorAssignment](https://github.com/AdityaNow/NavtorAssignment)

I am happy to star yours!

**Reply by the1Riddle** [reply]
just did that, thanks

**Reply by ternera** [reply]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by AdityaNow** [reply]
Done, please star mine too - https://github.com/AdityaNow/NavtorAssignment/
Thank you

**Comment by fabricio-fn** [comment]
@peterbaccaro done!
